osu.Game/Overlays/BeatmapSet/Buttons/DownloadButton.cs

```diff
@@ -11,17 +13,24 @@ namespace osu.Game.Overlays.BeatmapSet.Buttons
 {
     public class DownloadButton : HeaderButton
     {
-        public DownloadButton(string title, string subtitle)
+        public DownloadButton(BeatmapSetInfo set, bool noVideo = false)
         {
             Width = 120;
 
+            string subtitle = string.Empty;
+
+            if (set.OnlineInfo.HasVideo)
+                subtitle = noVideo ? "without Video" : "with Video";
```

Comment:
Is `with Video` really necessary though? If it has no video you see no text at all. If it has a video you see `without Video` **and** `with Video` which makes it also visually hard to discern on the fly.
I'd suggest just labeling `without Video` on one of the buttons.

Reply:
tend to agree

Reply:
Imo it looks better with both having a subtitle, but it can be changed. Was just following osu!web design.